ocp4/profiles/coreos-ncp.profile

```diff
@@ -56,216 +60,210 @@ selections:
     #- partition_for_home
     #- partition_for_var
     #- mount_option_var_nodev
-    - partition_for_var_log
+    #- partition_for_var_log
     #- mount_option_var_log_nodev
     #- mount_option_var_log_nosuid
     #- mount_option_var_log_noexec
-    - partition_for_var_log_audit
+    #- partition_for_var_log_audit
     #- mount_option_var_log_audit_nodev
     #- mount_option_var_log_audit_nosuid
     #- mount_option_var_log_audit_noexec
 
     ### Services
     # sshd
-    - sshd_disable_root_login
-    - sshd_enable_strictmodes
-    - disable_host_auth
-    - sshd_disable_empty_passwords
-    - sshd_disable_kerb_auth
-    - sshd_disable_gssapi_auth
-    - var_sshd_set_keepalive=0
-    - sshd_set_keepalive
-    - sshd_enable_warning_banner
-    - sshd_rekey_limit
-
-    # Time Server
-    - chronyd_client_only
```

Comment:
@jhrozek Have the chronyd_ rules disappeared by purpose?

Reply:
I re-added them, thanks for catching that.